### ISSUE TITLE
Support for maven 3.9 in the docker images for mbt

### ISSUE THREAD

**Issue by tofcoder** [SAP/cloud-mta-build-tool#1047]
It would be nice to have docker images including maven 3.9 to be able to build applications with it as this version supports a new dependency collector (breadth-first collector) that speeds up builds considerably as POM downloads can be done in parallel.

**Comment by young-yang03** [comment]
@r-bhuvan and @tofcoder We will upgrade the Maven to 3.9.2 soon

**Comment by young-yang03** [comment]
@r-bhuvan and @tofcoder We have upgrade the Maver to 3.9.3, the new release version is MBT v1.2.24
